source/features/show-associated-branch-prs-on-fork.tsx

```diff
@@ -42,11 +45,11 @@ const getPullRequestsAssociatedWithBranch = cache.function(async (): Promise<Rec
 
 	const pullRequests: Record<string, PullRequest> = {};
 	for (const {name, associatedPullRequests} of repository.refs.nodes) {
-		const [prInfo] = associatedPullRequests.nodes;
+		const [prInfo] = associatedPullRequests.nodes as PullRequest[];
 		// Check if the ref was deleted, since the result includes pr's that are not in fact related to this branch but rather to the branch name.
 		const headRefWasDeleted = prInfo?.timelineItems.nodes[0]?.__typename === 'HeadRefDeletedEvent';
 		if (prInfo && !headRefWasDeleted) {
-			prInfo.state = prInfo.isDraft && prInfo.state === 'OPEN' ? 'Draft' : upperCaseFirst(prInfo.state);
+			prInfo.state = prInfo.isDraft && prInfo.state === ('OPEN' as 'Open') ? 'Draft' : upperCaseFirst(prInfo.state) as keyof typeof stateClass;
```

Comment:
I see now why it requires it this way: `prInfo` is now typed 👌

Reply:
I think it's easier if we don't use `upperCaseFirst` at all on this line
```suggestion
			prInfo.state = prInfo.isDraft && prInfo.state === 'OPEN' ? 'DRAFT' : prInfo.state;
```